demo

```diff
@@ -211,45 +233,72 @@ done
 
 
 echo "Creating a new user..."
-UPLOAD_ARTIFACT="true"
-RETRIES=0
-while :
-do
+UPLOAD_ARTIFACT=1
```

Comment:
Why the integer booleans, instead of idioms like `true` and `false`?

Reply:
No particular reason. Somewhat subjectively I would argue that in shell scripts integer style is more common, but I have no data to back that up. In any case, my point here was just to pick one and use only that, and the existing true/false variant was implemented incorrectly (it set the `true` value, but used empty value instead of `false`).

Reply:
Alright :)